stress/shell/src/main/java/alluxio/stress/cli/client/StressClientIOBench.java

```diff
@@ -77,6 +89,10 @@ public void prepare() throws Exception {
           "%s is a single-node client IO stress test, so it cannot be run in cluster mode without"
               + " flag '%s 1'.", this.getClass().getName(), BaseParameters.CLUSTER_LIMIT_FLAG));
     }
+    if (!mBaseParameters.mProfileAgent.isEmpty()) {
```

Comment:
Since this is now in base parameters, this block can be moved to `Benchmark`, right?